### ISSUE TITLE
brew install fails on Mac with M1 chip

### ISSUE THREAD

**Issue by macdonst** [get-woke/woke#139]
Could you please add arm64 in goreleaser, to build woke for M1?

The installation fails now with an error:

## Brew

```
brew install get-woke/tap/woke
==> Tapping get-woke/tap
Cloning into '/opt/homebrew/Library/Taps/get-woke/homebrew-tap'...
remote: Enumerating objects: 148, done.
remote: Counting objects: 100% (148/148), done.
remote: Compressing objects: 100% (74/74), done.
remote: Total 148 (delta 37), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (148/148), 15.95 KiB | 2.66 MiB/s, done.
Resolving deltas: 100% (37/37), done.
Error: Invalid formula: /opt/homebrew/Library/Taps/get-woke/homebrew-tap/Formula/woke.rb
formulae require at least a URL
Error: Cannot tap get-woke/tap: invalid syntax in tap!
```

## cURL

```
curl -sSfL https://git.io/getwoke | \
  bash -s -- -b /usr/local/bin
get-woke/woke crit platform darwin/arm64 is not supported.  Make sure this script is up-to-date and file request at https://github.com/get-woke/woke/issues/new
```

**Comment by caitlinelfring** [comment]
@macdonst  arm64 added to build architecture list has been released in `v0.15.0`. Would you mind testing for me? I don't have an M1 mac so would love confirmation. Thanks for requesting!

**Comment by macdonst** [comment]
@caitlinelfring just downloaded `woke` via brew on my M1 and it works like a charm. Thanks for making the change in goreleaser.
